src/main/java/io/gravitee/policy/json2json/configuration/JsonToJsonTransformationPolicyConfiguration.java

```diff
@@ -53,4 +53,31 @@ public boolean isOverrideContentType() {
     public void setOverrideContentType(boolean overrideContentType) {
         this.overrideContentType = overrideContentType;
     }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
```

Comment:
Could you use lombok to replace boilerplate code?

Reply:
I tried but it was breaking the integration test. But after another try, it needs to add `@NoArgsConstructor`. So I will use the Lombok builder instead ;)